### ISSUE TITLE
Nav offcanvas list view doesn't provide context on entry/focus

### ISSUE THREAD

**Issue by getdave** [WordPress/gutenberg#47010]
As reported in https://github.com/WordPress/gutenberg/issues/46939#issuecomment-1374652632 initially focusing the offcanvas list view doesn't provide suitable context about the purpose of the feature. 

The accessible Name of the List View should be Navigation Menu: %%NAV MENU NAME%%. Currently it's something about "Application" which is confusing.

**Comment by getdave** [comment]
Duplicate of https://github.com/WordPress/gutenberg/issues/47022
